content/shmem_team_split_strided.tex

```diff
@@ -59,6 +59,17 @@
 relative order with respect to the parent team.
 If a $stride$ value equal to 0 is passed to \FUNC{shmem\_team\_split\_strided},
 then the $size$ argument passed must be 1, or the behavior is undefined.
+When $stride$ is nonzero, a newly created team must only include \acp{PE} whose subsequent parent \ac{PE}
+values are either all increasing (for positive $stride$) or all decreasing
+(for negative $stride$).
+That is, \textit{wrap-around} with respect to the parent team's \ac{PE} values
+is not permitted.
+For example, the list of \acp{PE} in the parent team should not start at a high
+number and then continue to include \acp{PE} in the lower end of the parent
+team's \ac{PE} range.
+If the triplet provided to \FUNC{shmem\_team\_split\_strided} implies such a
```

Comment:
I would suggest moving this part of the change to the top (line 62).  The content between 62 and 69 would then clarify what is meant by this.

Reply:
that would also mean that "implies such a wrap-around" would be changed to "implies a wrap-around".

Reply:
Done!  Can you please review the change to see if I captured it well?  Thanks, @bobkierski